datahub-web-react/src/graphql/schemaBlame.graphql

```diff
@@ -0,0 +1,46 @@
+query getSchemaBlame($input: GetSchemaBlameInput!) {
+    getSchemaBlame(input: $input) {
```

Comment:
we can address this later.. but TBH it feels like this should be split into 2 calls:

1. Get all versions of the schema (version-ordered list of summary objects (version + timestamp) 
2. Get the specifics of the current version (invoked when you change the drop down) 

I wonder if there will be any perf concerns with fetching everything at once

Reply:
(Oh- actually it seems that we do this) - Then do we really need to fetch the array of "all versions" in this query?

Reply:
I think the type checker got made when I did this, but it's not any extra work to fetch this information.

Reply:
Will attempt to remove actually

Reply:
Done!